mlrun/frameworks/_ml_common/pkl_model_server.py

```diff
@@ -36,11 +37,15 @@ def predict(self, body: dict) -> list:
         Infer the inputs through the model using MLRun's PyTorch interface and return its output. The inferred data will
         be read from the "inputs" key of the request.
 
-        :param request: The request of the model. The input to the model will be read from the "inputs" key.
+        :param body: The request of the model. The input to the model will be read from the "inputs" key.
```

Comment:
looking at the super class, you should change the parameter name to `request` and not the docstring